packages/adminrouter/extra/src/includes/http/common-windows.conf

```diff
@@ -86,19 +86,12 @@
 
 client_max_body_size 1024M;
 
-# Define custom log format. Use the default 'combined'
-# format and append easily parsable timing information
-# for performance analysis, and other relevant debugging
-# information.
-log_format customformat '$remote_addr - $remote_user [$time_local] '
-    '"$request" $status $body_bytes_sent '
-    '"$http_referer" "$http_user_agent" '
-    'logtime_msec=$msec '
-    'upstream_addr=$upstream_addr '
-    'request_time=$request_time '
-    'upstream_response_time=$upstream_response_time '
-    'upstream_connect_time=$upstream_connect_time '
-    'upstream_header_time=$upstream_header_time';
+# Define custom log format.
+log_format customformat '$time_local $pid *$connection '
+    '<$remote_addr:$remote_port $http_user_agent $http_referer '
```

Comment:
Just a note: I was confused by `<` and `>` in this and assumed they were special characters.
* It seems to me that we want to indicate the log-line in this format `< some-log-line >`  - Is this correct? Will it help if we had both start-symbol `<` and and end-symbol `>` in the same line?